test/e2e/Tests/build-e2e-test.ps1

```diff
@@ -13,7 +13,7 @@ param(
     $SkipStorageEmulator,
 
     [Switch]
-    $SkipCosmosDBEmulator,
+    $StartCosmosDBEmulator,
```

Comment:
We don't have any dependency on Cosmos DB, so we can remove logic related to starting the emulator.

Reply:
Will address this in a separate PR